moto/elasticbeanstalk/models.py

```diff
@@ -42,12 +47,18 @@ def region(self) -> str:
 
 
 class FakeApplication(BaseModel):
-    def __init__(self, backend: "EBBackend", application_name: str):
+    def __init__(
+        self,
+        backend: "EBBackend",
+        application_name: str,
+    ):
         self.backend = weakref.proxy(backend)  # weakref to break cycles
         self.application_name = application_name
         self.environments: Dict[str, FakeEnvironment] = dict()
         self.account_id = self.backend.account_id
         self.region = self.backend.region_name
+        self.status = ""
+        self.terminate_env_by_force = False
```

Comment:
Let's remove this variable - it doesn't have any impact on how users interact with Moto, so there's no point in keeping it.

Reply:
The variable `status` was removed.